lib/ddtrace/registry.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'registry/registerable'
```

Comment:
should be a `require_relative`?

Reply:
I usually prefer relative paths, but there is more a matter of style I guess. I think `require_relative` tend to be more common in gems, specially when you're loading components that are associated.

Reply:
👍